Dockerfile

```diff
@@ -177,6 +179,10 @@ COPY --from=trtserver_caffe2 /opt/conda/lib/libmkl_intel_lp64.so /opt/tensorrtse
 COPY --from=trtserver_caffe2 /opt/conda/lib/libmkl_rt.so /opt/tensorrtserver/lib/
 COPY --from=trtserver_caffe2 /opt/conda/lib/libmkl_vml_def.so /opt/tensorrtserver/lib/
 
+# LibTorch library
+COPY --from=trtserver_caffe2 /opt/conda/lib/python3.6/site-packages/torch/lib/libtorch.so* \
```

Comment:
How many libtorch libraries are there? Why the wildcard?

Reply:
libtorch.so and libtorch.so.1

Reply:
You don't want to copy both of those since one is just a link and then you will end up with two copies of the file. Instead you want to copy one and create the other as a link. You can do this in the docker file, ask Guan for help if you need it since it has done it several times.

Reply:
Sure :)